aim/storage/structured/sql_engine/models.py

```diff
@@ -85,3 +93,45 @@ def __init__(self, name):
     def validate_color(self, _, color):
         # TODO: [AT] add color validation
         return color
+
+
+class Note(Base):
+    __tablename__ = 'note'
+    __table_args__ = (
```

Comment:
do we need this constraint? why restrict someone to add the same note (comment) multiple times?

Reply:
@alberttorosyan its like having `new folder` and `new folder` on your desktop :D how would you distinguish ?